src/carbon-calculation/interfaces/EC3Configurations.ts

```diff
@@ -97,17 +97,45 @@ export interface EC3ConfigurationCollection {
 }
 
 /**
- * Properties of the EC3 Configuration to be created.
+ * Properties of the EC3 Configuration to be created. Part of Carbon Calculation EC3 API report schema.
  * @export
- * @interface EC3ConfigurationCreate
+ * @interface EC3ReportConfigurationCreate
  */
-export interface EC3ConfigurationCreate {
+export interface EC3ReportConfigurationCreate extends EC3ConfigurationCreate {
   /**
    * Id of the Report.
    * @type {string}
    * @memberof EC3ConfigurationCreate
    */
   reportId: string;
+}
+
+/**
+ * Properties of the EC3 Configuration to be created. Part of Carbon Calculation EC3 API extraction schema.
+ * @export
+ * @interface EC3ReportConfigurationCreate
+ */
+export interface EC3ExtractionConfigurationCreate extends EC3ConfigurationCreate {
+  /**
+   * Id of the iTwin.
+   * @type {string}
+   * @memberof EC3ConfigurationCreate
```

Comment:
Can you fix the mismatched tag `@memberof`? Might as well for the entire file. The interface name should match exactly.